modules/nvidia_plugin/tests/functional/shared_tests_instances/single_layer_tests/ov_unsymmetrical_comparer.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2021-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common_test_utils/test_common.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "openvino/core/model.hpp"
+
+namespace ov {
+namespace test {
+
+template <class BaseLayerTest>
+class UnsymmetricalComparer : public BaseLayerTest, virtual public ov::test::SubgraphBaseTest {
+protected:
+    ov::Tensor generate_input(const ov::element::Type element_type, const ov::Shape& shape, int seed) const {
+        return ov::test::utils::create_and_fill_tensor(element_type, shape, 10, 0, 1, seed);
```

Comment:
Could we extend generation here to avoid of overload the method?
https://github.com/openvinotoolkit/openvino/blob/d6c2a10b383dc63563896486906e127f60b5a194/src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp#L51

Reply:
Moved to separate ticket 122480